src/main/java/no/nav/pensjon/selvbetjeningopptjening/security/RequestContext.java

```diff
@@ -0,0 +1,130 @@
+package no.nav.pensjon.selvbetjeningopptjening.security;
+
+import io.jsonwebtoken.Claims;
+import no.nav.pensjon.selvbetjeningopptjening.config.AppIds;
+import no.nav.pensjon.selvbetjeningopptjening.security.oauth2.TokenInfo;
+import no.nav.pensjon.selvbetjeningopptjening.security.token.EgressTokenSupplier;
+import no.nav.pensjon.selvbetjeningopptjening.security.token.RawJwt;
+
+import static java.util.Objects.requireNonNull;
+import static no.nav.pensjon.selvbetjeningopptjening.security.jwt.ClaimsUtil.getInternalUserId;
+import static org.springframework.util.StringUtils.hasText;
+
+/**
+ * The context (user info etc.) of the request processing thread.
+ * Important: Use try-with-resources when using this auto-closeable class. This will ensure that the context
```

Comment:
Antar dette da kun gjelder de metodene som gjør instance.set(context)? (og ikke getEgressToken for eksempel)

Reply:
Ja, alt som medfører `new RequestContext`.